### ISSUE TITLE
Accessibility: ipython.org

### ISSUE THREAD

**Issue by Damian-chibuike** [jupyterhub/outreachy#95]
### What page is this for?

https://ipython.org/

### WAVE accessibility report

![IPython screenshot](https://user-images.githubusercontent.com/63458545/196708256-ccad6aa1-9391-495c-ad00-095ecf220d1c.png)

- The language of the document was not specified.
- The text-to-background colour contrast ratio was low, 4.07:1 was used instead of the standard ratio of 4.5:1
- Insufficient alternative text for the logo icon, it is succinct but does not expressly describe the icon for better understanding for impaired readers.
- Alternative text was not added to the IPythone Notebook viewer image, and this will affect screen reader users negatively, as it provides an alternative for the image.
- Full justification creates 'rivers of white' that flows through text and hinders readability, caused by the use of <p>, <div> or <td>

### The HTML element to be changed

```html
nill

<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Install
</a>

<img class="logo" src="_static/IPy_header.png" alt="Logo">

<img src="_static/nbviewer-thm-6.png" title="IPython Notebook Viewer">

<p>
Several of the authors of IPython are connected with academic and scientific research, so it is important for us to be able to show the impact of our work in other projects and fields.
</p>
```

### Your proposed HTML (or CSS), after change

```html
<html lang="en">

<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: #A1A1A1; background-color: #000000;">
Install
</a>

<img class="logo" src="_static/IPy_header.png" alt="IPython Interactive computing logo">

<img src="_static/nbviewer-thm-6.png" title="IPython Notebook Viewer" alt="IPython Notebook Viewer">

<html>
<head>
<style>
p {
  color: black;
  text-indent: 30px;
  text-transform: uppercase;
}
</style>
</head>
<body>

<p>Several of the authors of IPython are connected with academic and scientific research, so it is important for us to be able to show the impact of our work in other projects and fields..</p>

</body>
</html>
```

### More info

The first correction of `<htmi lang = "en">` was not included on the original page, but introduced in my proposal

 Full justification creates 'rivers of white' that flows through text and hinders readability, caused by the use of <p>, <div> or <td>. Hence it is better to use CSS to style paragraphs.

p {
  color: black;
  text-indent: 30px;
  text-transform: uppercase;
}
</style>
</head>
<body>

<p>
</p>

</body>
</html>

**Comment by minrk** [comment]
Thanks for the suggestions! Can I ask why you added `text-transform: uppercase`?

**Comment by Damian-chibuike** [comment]
Hello @minrk, the `text-transform: uppercase` was a mistake on my end, sorry about that.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
